omero/sysadmins/public.rst

```diff
@@ -138,3 +139,132 @@ authentication. For example a direct link to image will look as follows::
 For more details about how to create an OMERO session see
 :doc:`server-side session </developers/Server/Sessions>` or
 use the :doc:`command line interface </users/cli/sessions>` to create one.
+
+.. _hosting_data_example:
+
+Complete workflow for hosting data for a publication
+----------------------------------------------------
+
+Putting the pieces of this puzzle together, the following describes the steps
+of a complete workflow for using OMERO to host public data associated with a
+publication. It is illustrated using an example publication from the Swedlow
+lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_ with the
```

Comment:
update link to http://rsob.royalsocietypublishing.org/content/7/11/170099

Reply:
Isn't this precisely what DOIs are for?

Reply:
it is but elsewhere in the doc we use the "final" link

Reply:
I can't get that to work. Minimally those URLs use different subpaths: `/content/royopenbio` for jpegs vs. just `/content` for the target of the DOI.

Reply:
ignore in that case

Reply:
Looking for a "component DOI" now: http://blogs.plos.org/mfenner/2011/03/26/direct-links-to-figures-and-tables-using-component-dois/

Reply:
@joshmoore - can we link ["launching web viewer"](https://github.com/openmicroscopy/ome-documentation/pull/1830/files#diff-f7e55dec15373a4b0fe85781b7e29929R13) back to this page?

Reply:
Does the link at the top count? (see https://github.com/openmicroscopy/ome-documentation/pull/1830/commits/c6cd42c9ae2602e4a2279522e86c6f346faeca2b)

Reply:
A "Ref" with link directly after [the sentence "please remember"](https://github.com/openmicroscopy/ome-documentation/commit/c6cd42c9ae2602e4a2279522e86c6f346faeca2b#diff-b0666ac64875d77a655a2a545b172593R75) would be nicer imho.

Reply:
https://github.com/openmicroscopy/ome-documentation/pull/1830/commits/f9329233b51603d5d47d5bd8e407a1b937d81a14 ?